stdlib/list.gr

```diff
@@ -222,15 +235,18 @@ let part = (count, list) => {
 }
 
 let rotate = (count, list) => {
-  let (beginning, end) = if (count >= 0) part(count, list) else part(length(list) + count, list)
+  let (beginning, end) = if (count >= 0) part(count, list)
+  else part(length(list) + count, list)
```

Comment:
We want to indent this one indent if assigned to the variable?
```suggestion
  let (beginning, end) = if (count >= 0) part(count, list)
    else part(length(list) + count, list)
```

Reply:
Fixed.